beacon_node/http_api/src/lib.rs

```diff
@@ -3947,7 +4034,9 @@ pub fn serve<T: BeaconChainTypes>(
         .boxed()
         .uor(
             warp::post().and(
-                post_beacon_blocks
+                warp::header::exact("Content-Type", "application/octet-stream")
+                    .and(post_beacon_blocks_ssz.uor(post_beacon_blocks_v2_ssz))
```

Comment:
```suggestion
                    // Routes which expect `application/octet-stream` go within this `and`.
                    .and(post_beacon_blocks_ssz.uor(post_beacon_blocks_v2_ssz))
```

Just a little helper for future devs.